kitsune/wiki/models.py

```diff
@@ -857,8 +865,6 @@ class Revision(ModelBase, SearchMixin):
     # this is simply the latest of the default locale's revs as of that time.
     # Used to determine whether localizations are out of date.
     based_on = models.ForeignKey('self', null=True, blank=True)
-    # TODO: limit_choices_to={'document__locale':
```

Comment:
I don't think you have handled this TODO item in this PR. Then again, I don't really know what it is talking about. Do you?